Automattic/wp-calypso
# Sharing: Add Photos from Google service

Adds a sharing service definition for Google Photos. Note this just adds the service to the Sharing page and doesn't provide any Google Photos functionality.

Also note that the service is referred to as the 'Photos from Google' service, at the request of Google not to use the term 'Google Photos' to avoid confusion with its use of the Picasa API. It is internally referred to as Google Photos to reduce impact when/if we stop using the Picasa API.

The service has a custom logo that does not exist in SocialLogos, as per the other services. This is because the Google Photos logo requires multiple colours and SocialLogos strips all colours. An [issue has been created](https://github.com/Automattic/social-logos/issues/46) in the SocialLogos package, but until then we will need to use the custom logo included here.

#### Testing

1. Go to Sharing page for a site when logged in with an Automattic user
1. Verify that Photos from Google appears in the list with a screenshot when expanded

<img width="744" alt="sharing" src="https://cloud.githubusercontent.com/assets/1277682/25746917/79c5d088-319d-11e7-878c-a1b0bd5b5f36.png">

3. Verify all other services appear as expected
1. Click 'Connect' on the service and authorize a connection
1. Verify the service is connected and shows your Google account profile picture when expanded

<img width="341" alt="connected" src="https://cloud.githubusercontent.com/assets/1277682/25747545/b81bcbce-319f-11e7-8ae3-d036c9dd96ed.png">

6. Disconnect the service and verify it stays disconnected
